### PR TITLE
Ability to ssh to github actions runnner.

### PR DESCRIPTION
**Pull request by sanebay**
ssh via tmate will close the VM, if UT's fail. so put a sleep instead and build manually.

### CODE REVIEW

File: .github/workflows/build_dependencies.yml [hunk 1]
```diff
@@ -83,6 +83,7 @@ on:
 jobs:
   BuildHomestoreDeps:
     runs-on: ${{ inputs.platform }}
+    timeout-minutes: 1440
```
Comment (sanebay) [inline .github/workflows/build_dependencies.yml:86]: From  net, to add this to keep the VM for longer time.

File: .github/workflows/build_dependencies.yml [hunk 2]
```diff
@@ -206,10 +207,18 @@ jobs:
...
+      uses: mxschmitt/action-tmate@v3
+      with:
+        limit-access-to-actor: true
+        detached: true
+
     - name: Create and Test Package
       run: |
         sanitize=$([[ "${{ inputs.tooling }}" == "Sanitize" ]] && echo "True" || echo "False")
         pre=$([[ "${{ inputs.build-type }}" != "Debug" ]] && echo "-o sisl:prerelease=${{ inputs.prerelease }}" || echo "")
+        sudo rm -rf /usr/local/lib/android/
```
Comment (sanebay) [inline .github/workflows/build_dependencies.yml:221]: For sanitize build, VM is too low on space. By removing this unwanted folder, we get some 10G extra.

**Review by szmyd — APPROVED** [review]
makes sense; how is it used?

**Comment by sanebay** [comment]
> makes sense; how is it used?
In actions, there is a tab for Setup tmate session which gives details about the ssh connection.

image

**Pull request merged**